browser/brave_rewards/extension_rewards_service_observer.cc

```diff
@@ -104,8 +104,8 @@ void ExtensionRewardsServiceObserver::OnPanelPublisherInfo(
 
 void ExtensionRewardsServiceObserver::OnFetchPromotions(
     RewardsService* rewards_service,
-    const uint32_t result,
-    const std::vector<brave_rewards::Promotion>& list) {
+    const ledger::Result result,
+    const ledger::PromotionList list) {
```

Comment:
I think this should still be a const reference everywhere (i.e., it still needs the `&`).

Reply:
fixed